### PR TITLE
evilify VC

### PR DESCRIPTION
**Pull request by dvzubarev**
It may solve #3687

### CODE REVIEW

File: layers/+source-control/version-control/README.org [hunk 1]
```diff
@@ -59,12 +65,90 @@ one over the other:
...
+Navigation and interaction commands the VC Directory buffer,:
+| Key Bindings    | Description        |
+|-----------------+--------------------|
+| ~j~ or ~M-n~    | next file          |
+| ~k~ or ~M-p~    | previous file      |
+| ~gj~ or ~TAB~   | next directory     |
+| ~gk~ or ~S-TAB~ | previous directory |
+| ~m~             | mark a file        |
+| ~u~             | unmark a file      |
+** Commit message editing buffer
```
Comment (robbyoconnor) [inline layers/+source-control/version-control/README.org]: Spacing. Why is there no spacing?

Comment (dvzubarev) [inline layers/+source-control/version-control/README.org]: fixed

Comment (robbyoconnor) [inline layers/+source-control/version-control/README.org]: :100:

**Comment by d12frosted** [comment]
@TheBB, @syl20bnr, @bmag and other vim/hybrid users. What do you think about it? 😸

**Comment by nixmaniack** [comment]
I generally use `vc-annotate` (`C-x v g`) which isn't evilified at the moment in `hybrid`. I would at least 👍 for that.

**Comment by d12frosted** [comment]
@nixmaniack have you tried this PR? 😸

**Comment by nixmaniack** [comment]
@d12frosted Actually I just use `vc-annotate` so that's the only useful part I tested from this PR which works as expected.
```
(evilified-state-evilify vc-annotate-mode vc-annotate-mode-map
  "J" 'vc-annotate-next-revision
  "K" 'vc-annotate-prev-revision
  "L" 'vc-annotate-show-log-revision-at-line
  "H" 'vc-annotate-toggle-annotation-visibility
  "a" 'vc-annotate-revision-at-line
  "p" 'vc-annotate-revision-previous-to-line)
```
Otherwise `magit` has all the features I need so didn't try other things.

**Comment by d12frosted** [comment]
@nixmaniack I see. Anyway, thanks for checking! 😸

**Comment by bmag** [comment]
Haven't had a need for VC yet (Magit and aHg covered me well so far) and didn't try out the PR, but the code looks good :+1:

**Comment by syl20bnr** [comment]
Thank you ! 💜 💜 💜 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**